src/_pytest/python.py

```diff
@@ -216,18 +216,6 @@ def pytest_pycollect_makemodule(path, parent):
     return Module(path, parent)
 
 
-def pytest_ignore_collect(path, config):
```

Comment:
Yeah this does not really seem necessary, `main.pytest_ignore_collect` already seems to do the exactly same thing anyway.

Reply:
Initially I copy/pasted the Package class from Session because they both serve very similar functions, and then realized not everything was needed.